eland/tests/ml/test_imported_ml_model_pytest.py

```diff
@@ -358,10 +383,38 @@ def test_lgbm_regressor(self, compress_model_definition):
             overwrite=True,
             es_compress_model_definition=compress_model_definition,
         )
+        # Get some test results
+        test_data = [[0.1, 0.2, 0.3, -0.5, 1.0], [1.6, 2.1, -10, 50, -1.0]]
+        check_prediction_equality(es_model, regressor, test_data)
 
-        es_results = es_model.predict(test_data)
+        # Clean up
+        es_model.delete_model()
 
-        np.testing.assert_almost_equal(test_results, es_results, decimal=2)
+    @requires_lightgbm
+    @pytest.mark.parametrize("booster", ["gbdt", "rf", "dart", "goss"])
```

Comment:
I don't know a lot here but is it possible to move this `parametrize()` call to the above test suite or if that'd be too much effort to test all permutations? I don't know much about LightGBM so I'm unsure if this request even makes sense :)

Reply:
will move it